tests/test_fusor.py

```diff
@@ -766,6 +766,7 @@ async def test_transcript_segment_element(
     assert tsg[0].model_dump() == expected.model_dump()
 
     # MANE
+    # help wanted!!! I can't get this test to work and I can't figure out why - this is a valid start, but CST returns no data
```

Comment:
commenting here to help draw attention - this test fails, and I can't figure out why because it is passed a valid coordinate

`WARNING  cool_seq_tool.mappers.exon_genomic_coords:exon_genomic_coords.py:742 Unable to find mane data for NC_000011.10 with position 9576093 on gene WEE1
WARNING  cool_seq_tool.mappers.exon_genomic_coords:exon_genomic_coords.py:88 Unable to find mane data for NC_000011.10 with position 9576093 on gene WEE1
WARNING  cool_seq_tool.mappers.exon_genomic_coords:exon_genomic_coords.py:88 Unable to find mane data for NC_000011.10 with position 9576093 on gene WEE1
`

Reply:
make a new issue imo